Silence initialization of giac

tornaria opened the pull request:
When sage.libs.giac.giac is imported for the first time, some noise is output to stderr. This is annoying and complicates doctesting as `...` has to be added to the _first_ time giac is used.

This output comes from the line

    Pygen('add_language(1)').eval()  # Add the french keywords in the giac library language.

which doesn't seem to be necessary.

This commit removes the line, and fixes a few doctests which where expecting some output when using libgiac for the first time.

Also label `# long time` a doctest in `src/sage/calculus/functional.py` which takes 6-7 seconds. Doing this in the first place motivates the current change as the initialization noise would happen in different tests depending on whether `--long` or not is used.

Same for a doctest in `src/sage/calculus/tests.py`, and one in `src/sage/symbolic/integration/integral.py`. A test in `src/sage/libs/giac/giac.pyx` was marked `# random` but after this fix it seems to work ok.

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [x] There are existing tests covering the changes.

tornaria committed:
Silence initialization of giac

When sage.libs.giac.giac is imported for the first time, some noise is
output to stderr. This is annoying and complicates doctesting as `...`
has to be added to the _first_ time giac is used.

This output comes from the line

    Pygen('add_language(1)').eval()  # Add the french keywords in the giac library language.

which doesn't seem to be necessary.

This commit removes the line, and fixes a few doctests which where
expecting some output when using libgiac for the first time.

Also label `# long time` a doctest in `src/sage/calculus/functional.py`
which takes 6-7 seconds. Doing this in the first place motivates the
current change as the initialization noise would happen in different
tests depending on whether `--long` or not is used.

Same for a doctest in `src/sage/calculus/tests.py`, and one in
`src/sage/symbolic/integration/integral.py`. A test in
`src/sage/libs/giac/giac.pyx` was marked `# random` but after this fix
it seems to work ok.

kiwifb reviewed (approved):
The bots are happy and everything that needed sorting has been. Let's move on.

tornaria committed:
Fix doctest so it works with old and new giac.

This `...` corresponds to a warning in the version of giac bundled with
sage, but the warning is removed in current version of giac (tested with
1.9.0-43).

Before this `...` was ok since it was matching giac initialization, but
after the last commit this is no longer the case.

Placing the `...` in the same line as the output makes it optional.
